### ISSUE TITLE
General: More assets!

### ISSUE THREAD

**Issue by Germlord** [EdwardDunn/Platform-Game#171]
I would like to propose a general request for more visual assets. Currently, we have a fairly limited set of visual resources to work with. Here are things we could use more of:

-Platforms--despite the name of this game, we lack objects, aside from enemies, that the player can actually jump on.

-Backgrounds--the current backgrounds start with a fairly high ground level, making more complex jumping difficult. We could probably use backgrounds that allow for more complex movement--starting lower, not having a ground level at all, etc.

And, of course, anything else you can think of that might be useful. Please let me know what you think!

**Comment by EdwardDunn** [comment]
@Germlord I totally agree. More obstacles and objects to jump on would really add to the game, a bit more like the classic Mario games.
